stripe/src/main/java/com/stripe/android/model/PaymentMethod.kt

```diff
@@ -628,6 +637,13 @@ data class PaymentMethod internal constructor(
         @JvmField val last4: String?
     ) : StripeModel
 
+    @Parcelize
```

Comment:
Should this and AuBecsDebit be documented?

Reply:
Yes I'll add that in a follow-up PR